Assets/Scripts/Managers/CultistManager.cs

```diff
@@ -44,12 +38,15 @@ void Update()
 
     public void SpawnCultist()
     {
-        if (Random.Range(1, 3) == 1)
-            _spawnPosition = new Vector2(12.5f, -2.07f);
-        else
-            _spawnPosition = new Vector2(8.5f, -2.07f);
+        //between 7 & 13
+        //Altar pos is 10, most likely will have to add better way of detecting it
+        int minimum = 7;
+        int maximum = 13;
+        int spawnXCoordinates = Random.Range(minimum, maximum + 1);
+
+        Vector2 spawnPosition = new Vector2(spawnXCoordinates, -2.07f);
```

Comment:
magic number -2.07f, maybe make serializefield so we can edit it in inspector (if something about the world/map changes)